## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :x: [more...](#8af2ad5f-7b84-4007-a359-eca28799ee61) |
| mint-fs.sh | :x: [more...](#7a9b694c-228e-4b02-b80e-418de9794f80) |
| mint-gateway-s3.sh | :x: [more...](#8860a14a-b5a5-41b5-b5b2-16cb366eb869) |
| mint-erasure.sh | :x: [more...](#b24adc36-16df-43f7-9811-ab6386492d2b) |
| mint-dist-erasure.sh | :x: [more...](#749edc0d-361c-4d20-bb43-ebc7d7f61d8f) |
| mint-gateway-nas.sh | :x: [more...](#cee88e32-8962-47d7-a06e-3d6f64c87104) |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#9f36da9a-499d-482e-89f6-18ab439c8ef1) |
| mint-pools.sh | :x: [more...](#4ec8e36c-8bb0-4541-8a8b-46a26377a289) |
<a name="8af2ad5f-7b84-4007-a359-eca28799ee61"></a>
### 14589-0da59c5/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s2:30401
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 minutes and 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 57 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 26 seconds
(8/15) Running minio-dotnet tests ... done in 51 seconds
(9/15) Running minio-go tests ... done in 1 minutes and 21 seconds
(10/15) Running minio-java tests ... FAILED in 7 seconds
{
  "name": "minio-java",
  "function": "listObjects()",
  "args": "[bucket, prefix]",
  "duration": 5,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = SlowDown, message = A timeout exceeded while waiting to proceed with the request, please reduce your request rate, bucketName = null, objectName = null, resource = /minio-java-test-3e11tpn, requestId = 16DDF03F734D967A, hostId = 72725cbf-cb20-4420-90d8-908bcd54d25a)\nrequest={method=PUT, url=http://minio-k8s2:30401/minio-java-test-3e11tpn, headers=Host: minio-k8s2:30401\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004633Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 324\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nX-Amz-Request-Id: 16DDF03F734D967A\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:46:33 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.MinioClient.makeBucket(MinioClient.java:1199), FunctionalTest.testListObjects(FunctionalTest.java:1390), FunctionalTest.listObjects(FunctionalTest.java:1432), FunctionalTest.runObjectTests(FunctionalTest.java:3743), FunctionalTest.runTests(FunctionalTest.java:3781), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(10/15) Running minio-js tests ... done in 1 minutes and 3 seconds
(11/15) Running minio-py tests ... done in 39 minutes and 22 seconds
(12/15) Running s3cmd tests ... done in 26 seconds
(13/15) Running s3select tests ... done in 4 seconds
(14/15) Running versioning tests ... done in 4 minutes and 35 seconds

Executed 14 out of 15 tests successfully.
```
<a name="b24adc36-16df-43f7-9811-ab6386492d2b"></a>
### 14589-0da59c5/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s6:31515
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 minutes and 1 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 43 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... done in 1 minutes and 51 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 28 seconds
FAILED in 36 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate)) ---> System.AggregateException: One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate) ---> Minio.Exceptions.UnexpectedMinioException: MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate
   at Minio.MinioClient.ParseErrorFromContent(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 538
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 431
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 605
   at Minio.MinioClient.ExecuteTaskCoreAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 405
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, Dictionary`2 sseHeaders, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 532
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, ServerSideEncryption sse, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 301
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 806
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2048
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 137
(8/15) Running minio-go tests ... done in 56 seconds
(9/15) Running minio-java tests ... FAILED in 5 seconds
{
  "name": "minio-java",
  "function": "listObjects()",
  "args": "[bucket, prefix, recursive, 1050 objects]",
  "duration": 2006,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = BucketNotEmpty, message = The bucket you tried to delete is not empty, bucketName = minio-java-test-1kco904, objectName = null, resource = /minio-java-test-1kco904, requestId = 16DDF04AA2F22473, hostId = 5b838412-bd30-4bd1-b3a6-26a83916b5d1)\nrequest={method=DELETE, url=http://minio-k8s6:31515/minio-java-test-1kco904, headers=Host: minio-k8s6:31515\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004722Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=409, headers=Accept-Ranges: bytes\nContent-Length: 328\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nVary: Accept-Encoding\nX-Amz-Request-Id: 16DDF04AA2F22473\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:47:22 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.S3Base.execute(S3Base.java:440), io.minio.S3Base.executeDelete(S3Base.java:745), io.minio.MinioClient.removeBucket(MinioClient.java:1592), FunctionalTest.testListObjects(FunctionalTest.java:1418), FunctionalTest.listObjects(FunctionalTest.java:1457), FunctionalTest.runObjectTests(FunctionalTest.java:3743), FunctionalTest.runTests(FunctionalTest.java:3781), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(9/15) Running minio-js tests ... FAILED in 1 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 7,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(9/15) Running minio-py tests ... done in 20 minutes and 55 seconds
(10/15) Running s3cmd tests ... done in 27 seconds
(11/15) Running s3select tests ... done in 2 minutes and 2 seconds
(12/15) Running versioning tests ... done in 3 minutes and 1 seconds

Executed 12 out of 15 tests successfully.
```
<a name="4ec8e36c-8bb0-4541-8a8b-46a26377a289"></a>
### 14589-0da59c5/mint-pools.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s7:32064
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 5 seconds
(5/15) Running awscli tests ... done in 1 minutes and 52 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 28 seconds
FAILED in 39 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate)) ---> System.AggregateException: One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate) ---> Minio.Exceptions.UnexpectedMinioException: MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate
   at Minio.MinioClient.ParseErrorFromContent(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 538
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 431
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 605
   at Minio.MinioClient.ExecuteTaskCoreAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 405
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, Dictionary`2 sseHeaders, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 532
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, ServerSideEncryption sse, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 301
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 806
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2048
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 137
(8/15) Running minio-go tests ... FAILED in 37 seconds
{
  "args": {
    "bucketName": "minio-go-test-2seydiwbcy4a5usr",
    "objectName": "test-object",
    "opts": "",
    "size": -1
  },
  "duration": 181,
  "error": "A timeout exceeded while waiting to proceed with the request, please reduce your request rate",
  "function": "PutObject(bucketName, objectName, reader,size,opts)",
  "message": "PutObjectStreaming failed",
  "name": "minio-go: testPutObjectStreaming",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... FAILED in 5 seconds
{
  "name": "minio-java",
  "function": "deleteBucketLifecycle()",
  "duration": 160,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = SlowDown, message = A timeout exceeded while waiting to proceed with the request, please reduce your request rate, bucketName = null, objectName = null, resource = /minio-java-test-3jac00f, requestId = 16DDF0280FFF0887, hostId = a5e14697-9f5f-4b10-9453-bd0ac2ee0cbd)\nrequest={method=DELETE, url=http://minio-k8s7:32064/minio-java-test-3jac00f, headers=Host: minio-k8s7:32064\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004453Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 324\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nX-Amz-Request-Id: 16DDF0280FFF0887\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:44:53 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.S3Base.execute(S3Base.java:440), io.minio.S3Base.executeDelete(S3Base.java:745), io.minio.MinioClient.removeBucket(MinioClient.java:1592), FunctionalTest.deleteBucketLifecycle(FunctionalTest.java:2840), FunctionalTest.runBucketTests(FunctionalTest.java:3729), FunctionalTest.runTests(FunctionalTest.java:3780), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(8/15) Running minio-js tests ... FAILED in 7 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 11,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(8/15) Running minio-py tests ... done in 49 minutes and 34 seconds
(9/15) Running s3cmd tests ... done in 25 seconds
(10/15) Running s3select tests ... done in 2 minutes and 4 seconds
(11/15) Running versioning tests ... done in 9 minutes and 7 seconds

Executed 11 out of 15 tests successfully.
```
<a name="cee88e32-8962-47d7-a06e-3d6f64c87104"></a>
### 14589-0da59c5/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s5:30935
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 minutes and 1 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 42 seconds
(4/15) Running aws-sdk-ruby tests ... done in 3 seconds
(5/15) Running awscli tests ... done in 1 minutes and 48 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 18 seconds
FAILED in 17 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate)) ---> System.AggregateException: One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate) ---> Minio.Exceptions.UnexpectedMinioException: MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate
   at Minio.MinioClient.ParseErrorFromContent(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 538
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 431
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 605
   at Minio.MinioClient.ExecuteTaskCoreAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 405
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, Dictionary`2 sseHeaders, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 532
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, ServerSideEncryption sse, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 301
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 806
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2048
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 137
(8/15) Running minio-go tests ... FAILED in 13 seconds
{
  "args": {
    "bucketName": "minio-go-test-2gcone2aptlmfps2",
    "objectName": "test-object",
    "opts": "",
    "size": -1
  },
  "duration": 151,
  "error": "A timeout exceeded while waiting to proceed with the request, please reduce your request rate",
  "function": "PutObject(bucketName, objectName, reader,size,opts)",
  "message": "PutObjectStreaming failed",
  "name": "minio-go: testPutObjectStreaming",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... done in 16 seconds
(9/15) Running minio-js tests ... FAILED in 14 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 7,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(9/15) Running minio-py tests ... done in 8 minutes and 39 seconds
(10/15) Running s3cmd tests ... done in 20 seconds
(11/15) Running s3select tests ... done in 2 minutes and 2 seconds
(12/15) Running versioning tests ... done in 0 seconds

Executed 12 out of 15 tests successfully.
```
<a name="8860a14a-b5a5-41b5-b5b2-16cb366eb869"></a>
### 14589-0da59c5/mint-gateway-s3.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s3:31951
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 minutes and 1 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... FAILED in 0 seconds
{
  "name": "aws-sdk-php",
  "function": "anonDeleteObjects ( array $params = [] )",
  "args": {
    "Bucket": "aws-sdk-php-623678632a223",
    "Object": "obj1"
  },
  "duration": "107",
  "status": "FAIL",
  "error": "Error executing \"CopyObject\" on \"http://minio-k8s3:31951/aws-sdk-php-623678632a223/obj1-copy2\"; AWS HTTP error: Client error: `PUT http://minio-k8s3:31951/aws-sdk-php-623678632a223/obj1-copy2` resulted in a `400 Bad Request` response:\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>XMinioBackendDown</Code><Message>Remote backend is unreachable (503  (truncated...)\n XMinioBackendDown (client): Remote backend is unreachable (503 Service Unavailable) - <?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>XMinioBackendDown</Code><Message>Remote backend is unreachable (503 Service Unavailable)</Message><Key>obj1-copy2</Key><BucketName>aws-sdk-php-623678632a223</BucketName><Resource>/aws-sdk-php-623678632a223/obj1-copy2</Resource><RequestId>16DDF0026558B995</RequestId><HostId>a22ae6b4-0119-44b2-b311-b07689ffb978</HostId></Error>"
}
(3/15) Running aws-sdk-ruby tests ... done in 4 seconds
(4/15) Running awscli tests ... done in 1 minutes and 49 seconds
(5/15) Running healthcheck tests ... done in 0 seconds
(6/15) Running mc tests ... FAILED in 7 seconds
{
  "name": "mc",
  "duration": "2045",
  "function": "test_get_object_multipart",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-28532 --quiet --no-color cp myminio/mc-test-bucket-16375/mc-test-object-2381 mc-test-object-2381.downloaded  >>> \nmc: <ERROR> Unable to guess the type of copy operation. Invalid arguments provided, please refer `mc <command> -h` for relevant documentation."
}
(6/15) Running minio-dotnet tests ... done in 23 seconds
(7/15) Running minio-go tests ... FAILED in 4 seconds
{
  "args": {
    "destination": {
      "Bucket": "minio-go-test-byvsjqig2onimd6k-copy",
      "Object": "hzvznrkvc5brdcpm02c2m1deqdzc4g-copy",
      "Encryption": null,
      "UserMetadata": null,
      "ReplaceMetadata": false,
      "UserTags": null,
      "ReplaceTags": false,
      "LegalHold": "",
      "Mode": "",
      "RetainUntilDate": "0001-01-01T00:00:00Z",
      "Size": 0,
      "Progress": null
    },
    "source": {
      "Bucket": "minio-go-test-byvsjqig2onimd6k",
      "Object": "hzvznrkvc5brdcpm02c2m1deqdzc4g",
      "VersionID": "",
      "MatchETag": "c1aca41c3c307f745c2b897b733d84f3",
      "NoMatchETag": "",
      "MatchModifiedSince": "2014-03-31T00:00:00Z",
      "MatchUnmodifiedSince": "0001-01-01T00:00:00Z",
      "MatchRange": false,
      "Start": 0,
      "End": 0,
      "Encryption": null
    }
  },
  "duration": 53,
  "error": "400 Bad Request",
  "function": "CopyObject(destination, source)",
  "message": "Stat failed",
  "name": "minio-go: testCopyObjectV2",
  "status": "FAIL"
}
(7/15) Running minio-java tests ... FAILED in 2 seconds
(7/15) Running minio-js tests ... FAILED in 8 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 183,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(7/15) Running minio-py tests ... FAILED in 2 minutes and 54 seconds
{
  "name": "minio-py:test_list_objects_with_1001_files",
  "status": "FAIL",
  "args": {
    "bucket_name": "minio-py-test-1ec31924-15a9-4cdd-ba6b-81fe3dc4b325",
    "object_name": "10b985dc-1f5c-4a03-bbd7-99ec59ecdad6_0 ~ 0_1000"
  },
  "message": "S3 operation failed; code: BucketNotEmpty, message: The bucket you tried to delete is not empty, resource: /minio-py-test-1ec31924-15a9-4cdd-ba6b-81fe3dc4b325, request_id: 16DDF04F56CEF244, host_id: a22ae6b4-0119-44b2-b311-b07689ffb978, bucket_name: minio-py-test-1ec31924-15a9-4cdd-ba6b-81fe3dc4b325",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 126, in _call_test\n    func(log_entry, *args, **kwargs)\n  File \"/mint/run/core/minio-py/tests.py\", line 1243, in test_list_objects_with_1001_files\n    _CLIENT.remove_bucket(bucket_name)\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 669, in remove_bucket\n    self._execute(\"DELETE\", bucket_name)\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 397, in _execute\n    return self._url_open(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 380, in _url_open\n    raise response_error\nminio.error.S3Error: S3 operation failed; code: BucketNotEmpty, message: The bucket you tried to delete is not empty, resource: /minio-py-test-1ec31924-15a9-4cdd-ba6b-81fe3dc4b325, request_id: 16DDF04F56CEF244, host_id: a22ae6b4-0119-44b2-b311-b07689ffb978, bucket_name: minio-py-test-1ec31924-15a9-4cdd-ba6b-81fe3dc4b325\n",
  "duration": 159851
}
(7/15) Running s3cmd tests ... FAILED in 1 minutes and 10 seconds
{
  "name": "s3cmd",
  "duration": "15532",
  "function": "test_sync_list_objects",
  "status": "FAIL",
  "error": "WARNING: Bucket is not empty. Removing all the objects from it first. This may take some time...\nWARNING: Retrying failed request: / (503 (SlowDown): A timeout exceeded while waiting to proceed with the request, please reduce your request rate)\nWARNING: Waiting 3 sec...\ndelete: 's3://s3cmd-test-bucket-7206/datafile-0-b'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-1-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-1-b'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-1-kB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-1.03-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-10-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-10-kB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-100-kB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-11-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-129-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-33-kB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-5-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-5243880-b'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-6-MB'\ndelete: 's3://s3cmd-test-bucket-7206/datafile-65-MB'\nERROR: S3 error: 409 (BucketNotEmpty): The bucket you tried to delete is not empty"
}
(7/15) Running s3select tests ... FAILED in 1 seconds
{
  "name": "s3select:test_csv_input_quote_char",
  "function": "select_object_content(bucket_name, object_name, request)",
  "args": {
    "bucket_name": "s3select-test-6f74386e-4abc-431c-8847-7f7708876d54"
  },
  "duration": 709,
  "message": "Test test_16 unexpectedly failed with: InternalError: A timeout exceeded while waiting to proceed with the request, please reduce your request rate",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/s3select/csv.py\", line 42, in test_sql_api\n    for d in data.stream(10*1024):\n  File \"/usr/local/lib/python3.8/dist-packages/minio/select.py\", line 431, in stream\n    if self._read() <= 0:\n  File \"/usr/local/lib/python3.8/dist-packages/minio/select.py\", line 393, in _read\n    raise MinioException(\nminio.error.MinioException: InternalError: A timeout exceeded while waiting to proceed with the request, please reduce your request rate\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"./tests.py\", line 53, in main\n    test_csv_input_custom_quote_char(client, log_output)\n  File \"/mint/run/core/s3select/csv.py\", line 117, in test_csv_input_custom_quote_char\n    test_sql_api(f'test_{idx}', client, bucket_name,\n  File \"/mint/run/core/s3select/csv.py\", line 47, in test_sql_api\n    raise ValueError(\nValueError: Test test_16 unexpectedly failed with: InternalError: A timeout exceeded while waiting to proceed with the request, please reduce your request rate\n",
  "status": "FAIL"
}
(7/15) Running versioning tests ... done in 0 seconds

Executed 7 out of 15 tests successfully.
```
<a name="9f36da9a-499d-482e-89f6-18ab439c8ef1"></a>
### 14589-0da59c5/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s4:30116
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 4 minutes and 3 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... done in 1 minutes and 57 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 24 seconds
FAILED in 45 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate)) ---> System.AggregateException: One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate) ---> Minio.Exceptions.UnexpectedMinioException: MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate
   at Minio.MinioClient.ParseErrorFromContent(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 538
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 431
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 605
   at Minio.MinioClient.ExecuteTaskCoreAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 405
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, Dictionary`2 sseHeaders, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 532
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, ServerSideEncryption sse, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 301
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 806
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2048
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 137
(8/15) Running minio-go tests ... done in 1 minutes and 12 seconds
(9/15) Running minio-java tests ... FAILED in 4 seconds
{
  "name": "minio-java",
  "function": "getBucketPolicy()",
  "duration": 8,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = SlowDown, message = A timeout exceeded while waiting to proceed with the request, please reduce your request rate, bucketName = null, objectName = null, resource = /minio-java-test-2ftvi9t, requestId = 16DDF06D2EA89582, hostId = 4457442d-879a-4bf4-b637-f0911d272633)\nrequest={method=PUT, url=http://minio-k8s4:30116/minio-java-test-2ftvi9t, headers=Host: minio-k8s4:30116\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004950Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 324\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nX-Amz-Request-Id: 16DDF06D2EA89582\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:49:50 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.MinioClient.makeBucket(MinioClient.java:1199), FunctionalTest.getBucketPolicy(FunctionalTest.java:2692), FunctionalTest.runBucketTests(FunctionalTest.java:3724), FunctionalTest.runTests(FunctionalTest.java:3780), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(9/15) Running minio-js tests ... done in 1 minutes and 4 seconds
(10/15) Running minio-py tests ... done in 51 minutes and 17 seconds
(11/15) Running s3cmd tests ... done in 24 seconds
(12/15) Running s3select tests ... done in 2 minutes and 4 seconds
(13/15) Running versioning tests ... done in 5 minutes and 5 seconds

Executed 13 out of 15 tests successfully.
```
<a name="7a9b694c-228e-4b02-b80e-418de9794f80"></a>
### 14589-0da59c5/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s7:30489
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 43 seconds
(4/15) Running aws-sdk-ruby tests ... done in 3 seconds
(5/15) Running awscli tests ... done in 48 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 18 seconds
FAILED in 36 seconds

Unhandled Exception: System.AggregateException: One or more errors occurred. (One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate)) ---> System.AggregateException: One or more errors occurred. (MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate) ---> Minio.Exceptions.UnexpectedMinioException: MinIO API responded with message=A timeout exceeded while waiting to proceed with the request, please reduce your request rate
   at Minio.MinioClient.ParseErrorFromContent(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 538
   at Minio.MinioClient.ParseError(IRestResponse response) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 431
   at Minio.MinioClient.HandleIfErrorResponse(IRestResponse response, IEnumerable`1 handlers, DateTime startTime) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 605
   at Minio.MinioClient.ExecuteTaskCoreAsync(IEnumerable`1 errorHandlers, IRestRequest request, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/MinioClient.cs:line 405
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, String uploadId, Int32 partNumber, Byte[] data, Dictionary`2 metaData, Dictionary`2 sseHeaders, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 532
   at Minio.MinioClient.PutObjectAsync(String bucketName, String objectName, Stream data, Int64 size, String contentType, Dictionary`2 metaData, ServerSideEncryption sse, CancellationToken cancellationToken) in /q/.q/sources/minio-dotnet/Minio/ApiEndpoints/ObjectOperations.cs:line 301
   at Minio.Functional.Tests.FunctionalTest.PutObject_Task(MinioClient minio, String bucketName, String objectName, String fileName, String contentType, Int64 size, Dictionary`2 metaData, MemoryStream mstream) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 806
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.FunctionalTest.RemoveObjects_Test2(MinioClient minio) in /mint/run/core/minio-dotnet/FunctionalTest.cs:line 2048
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Minio.Functional.Tests.Program.Main(String[] args) in /mint/run/core/minio-dotnet/Program.cs:line 137
(8/15) Running minio-go tests ... done in 39 seconds
(9/15) Running minio-java tests ... FAILED in 0 seconds
{
  "name": "minio-java",
  "function": "makeBucket()",
  "args": "[basic check]",
  "duration": 206,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = SlowDown, message = A timeout exceeded while waiting to proceed with the request, please reduce your request rate, bucketName = null, objectName = null, resource = /minio-java-test-1dih97j, requestId = 16DDF012027BA307, hostId = b93391fb-6c5c-43ff-9bcd-b25c61d98b11)\nrequest={method=DELETE, url=http://minio-k8s7:30489/minio-java-test-1dih97j, headers=Host: minio-k8s7:30489\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004318Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 324\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nX-Amz-Request-Id: 16DDF012027BA307\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:43:18 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.S3Base.execute(S3Base.java:440), io.minio.S3Base.executeDelete(S3Base.java:745), io.minio.MinioClient.removeBucket(MinioClient.java:1592), FunctionalTest.testBucketApi(FunctionalTest.java:469), FunctionalTest.testBucketApiCases(FunctionalTest.java:480), FunctionalTest.makeBucket(FunctionalTest.java:511), FunctionalTest.runBucketTests(FunctionalTest.java:3704), FunctionalTest.runTests(FunctionalTest.java:3780), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(9/15) Running minio-js tests ... FAILED in 0 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 5,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(9/15) Running minio-py tests ... done in 38 minutes and 42 seconds
(10/15) Running s3cmd tests ... done in 18 seconds
(11/15) Running s3select tests ... FAILED in 1 seconds
{
  "name": "s3select:test_csv_output_quote_char",
  "function": "select_object_content(bucket_name, object_name, request)",
  "args": {
    "bucket_name": "s3select-test-2d630537-fb21-4af4-97ba-b4ee075e167b"
  },
  "duration": 95,
  "message": "Test test_5 unexpectedly failed with: S3 operation failed; code: SlowDown, message: A timeout exceeded while waiting to proceed with the request, please reduce your request rate, resource: /s3select-test-2d630537-fb21-4af4-97ba-b4ee075e167b/574a944a-b5f3-41bd-bef5-531b9d5904ba, request_id: 16DDF23359C312EA, host_id: b93391fb-6c5c-43ff-9bcd-b25c61d98b11",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/s3select/csv.py\", line 39, in test_sql_api\n    data = client.select_object_content(bucket_name, object_name, sql_opts)\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 563, in select_object_content\n    response = self._execute(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 397, in _execute\n    return self._url_open(\n  File \"/usr/local/lib/python3.8/dist-packages/minio/api.py\", line 380, in _url_open\n    raise response_error\nminio.error.S3Error: S3 operation failed; code: SlowDown, message: A timeout exceeded while waiting to proceed with the request, please reduce your request rate, resource: /s3select-test-2d630537-fb21-4af4-97ba-b4ee075e167b/574a944a-b5f3-41bd-bef5-531b9d5904ba, request_id: 16DDF23359C312EA, host_id: b93391fb-6c5c-43ff-9bcd-b25c61d98b11\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"./tests.py\", line 57, in main\n    test_csv_output_custom_quote_char(client, log_output)\n  File \"/mint/run/core/s3select/csv.py\", line 174, in test_csv_output_custom_quote_char\n    test_sql_api(f'test_{idx}', client, bucket_name,\n  File \"/mint/run/core/s3select/csv.py\", line 47, in test_sql_api\n    raise ValueError(\nValueError: Test test_5 unexpectedly failed with: S3 operation failed; code: SlowDown, message: A timeout exceeded while waiting to proceed with the request, please reduce your request rate, resource: /s3select-test-2d630537-fb21-4af4-97ba-b4ee075e167b/574a944a-b5f3-41bd-bef5-531b9d5904ba, request_id: 16DDF23359C312EA, host_id: b93391fb-6c5c-43ff-9bcd-b25c61d98b11\n",
  "status": "FAIL"
}
(11/15) Running versioning tests ... done in 0 seconds

Executed 11 out of 15 tests successfully.
```
<a name="749edc0d-361c-4d20-bb43-ebc7d7f61d8f"></a>
### 14589-0da59c5/mint-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s5:30118
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 minutes and 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 43 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... done in 56 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 27 seconds
(8/15) Running minio-dotnet tests ... done in 42 seconds
(9/15) Running minio-go tests ... done in 1 minutes and 28 seconds
(10/15) Running minio-java tests ... FAILED in 1 seconds
{
  "name": "minio-java",
  "function": "bucketExists()",
  "args": "[object lock]",
  "duration": 102,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = SlowDown, message = A timeout exceeded while waiting to proceed with the request, please reduce your request rate, bucketName = null, objectName = null, resource = /minio-java-test-1jr9unk, requestId = 16DDF03F0B925476, hostId = 70a28eb6-d468-4659-bf50-6f80e4c9212e)\nrequest={method=PUT, url=http://minio-k8s5:30118/minio-java-test-1jr9unk, headers=x-amz-bucket-object-lock-enabled: true\nHost: minio-k8s5:30118\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.3.6\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220320T004632Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220320/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-bucket-object-lock-enabled;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 324\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nX-Amz-Request-Id: 16DDF03F0B925476\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Sun, 20 Mar 2022 00:46:32 GMT\n}\n >>> [io.minio.S3Base.execute(S3Base.java:645), io.minio.MinioClient.makeBucket(MinioClient.java:1199), FunctionalTest.testBucketApi(FunctionalTest.java:452), FunctionalTest.testBucketApiCases(FunctionalTest.java:491), FunctionalTest.bucketExists(FunctionalTest.java:591), FunctionalTest.runBucketTests(FunctionalTest.java:3705), FunctionalTest.runTests(FunctionalTest.java:3780), FunctionalTest.runEndpointTests(FunctionalTest.java:3880), FunctionalTest.main(FunctionalTest.java:4001)]"
}
(10/15) Running minio-js tests ... FAILED in 19 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 8,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:165:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:89:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:670:10) at endReadableNT (internal/streams/readable.js:1334:12) at processTicksAndRejections (internal/process/task_queues.js:82:21)"
}
(10/15) Running minio-py tests ... done in 23 minutes and 18 seconds
(11/15) Running s3cmd tests ... done in 24 seconds
(12/15) Running s3select tests ... done in 3 seconds
(13/15) Running versioning tests ... done in 9 minutes and 6 seconds

Executed 13 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

--- a/cmd/globals.go
+++ b/cmd/globals.go
@@ -190,7 +190,7 @@ var (
 	globalBucketTargetSys    *BucketTargetSys
 	// globalAPIConfig controls S3 API requests throttling,
 	// healthcheck readiness deadlines and cors settings.
-	globalAPIConfig = apiConfig{listQuorum: 3}
+	globalAPIConfig = apiConfig{listQuorum: -1}
 
 	globalStorageClass storageclass.Config
 	globalLDAPConfig   xldap.Config

